tests/basic/async_dr_test.go

```diff
@@ -149,6 +153,135 @@ var _ = Describe("{MigrateDeployment}", func() {
 	})
 })
 
+var _ = Describe("{MigrateDeploymentMetroAsync}", func() {
+	var testrailID = 297595
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/297595
+	var runID int
+	var kubeConfigWritten bool
```

Comment:
Should this be a global variable that is accessible to all test cases?

Reply:
Done, Thanks!